index.html

```diff
@@ -1,9 +1,106 @@
-NPM Clone Nifty Penguin Magic npm Enterprise Products Solutions Resources Docs Support Search Join Log In Build amazing
-things Essential JavaScript development tools that help you go to market faster and build powerful applications using
-modern open source code. See plans Join for free Bring the best of open source to your company npm is the tool used by
-over 11,000,000 JavaScript developers around the world. Your developers already use it. Your company depends on it.
-Create an Org and get more out of the tools your team already knows and loves. Zero configuration Create an org, add
-your team, and start collaborating. Nothing to configure, set up, or manage. Team management Control who has access to
-what modules within your team namespace using straightforward team management capabilities. Familiar features npm Orgs
-has 100% parity with all the public npm registry features your developers already use. npm audit Enjoy the security
-auditing features built into the npm client, a zero-friction way to make open source software safer. Create an Org
+<!DOCTYPE html>
+<html>
+    <head><!-- index.html -->
+        <head>
+          ...
+          <link rel="stylesheet" href="./style.css" />
+        </head>
+ <body>       
+<header>
+  <div>
+    <div>
+      <img class="blackHeart" src="/home/sophiepa/ironhack/prework/labs/lab-html-exercise/images/black-heart.png" alt="black heart" />
```

Comment:
When you define `src` for an `<img/>` **always use a path to the image relative to your index.html file**. The value you gave it right now only shows the images from your computer's perspective, you should have given it `images/black-heart.png` or `./images/black-heart.png` instead.  Check[ this](https://www.computerhope.com/issues/ch001708.htm) if you want more details on that.